<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usersPage</class>
 <widget class="QMainWindow" name="usersPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startGameButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color: rgb(42, 33, 48);
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>start the game</string>
    </property>
   </widget>
   <widget class="QLabel" name="user1Label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;</string>
    </property>
    <property name="text">
     <string>first user</string>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundUsersLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/mainbg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="user2LineEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: white;
padding: 4px;
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>player 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="user2Label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;</string>
    </property>
    <property name="text">
     <string>second user</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user1LineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: white;
padding: 4px;
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>player 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="user1errorLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 133, 133);</string>
    </property>
    <property name="text">
     <string>please enter the first user info</string>
    </property>
   </widget>
   <widget class="QLabel" name="user2errorLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 133, 133);</string>
    </property>
    <property name="text">
     <string>please enter the second user info</string>
    </property>
   </widget>
   <zorder>backgroundUsersLabel</zorder>
   <zorder>startGameButton</zorder>
   <zorder>user1Label</zorder>
   <zorder>user2LineEdit</zorder>
   <zorder>user2Label</zorder>
   <zorder>user1LineEdit</zorder>
   <zorder>user1errorLabel</zorder>
   <zorder>user2errorLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
